packages/react/src/components/Comments/Comment.tsx

```diff
@@ -1,20 +1,16 @@
 "use client";
 
-import {
-  CommentData,
-  ThreadData,
-  mergeCSSClasses,
-  getDefaultEmojiPickerItems,
-  DefaultGridSuggestionItem,
-} from "@blocknote/core";
+import { CommentData, ThreadData, mergeCSSClasses } from "@blocknote/core";
+import data from "@emoji-mart/data";
```

Comment:
iirc this is a large package that we might only want to load when necessary (like we do for the regular emoji picker)